supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc

```diff
@@ -997,14 +979,24 @@ Use the name of the tab when you refer to it, for example, "the *Storage* tab".
 
 [[rpm]]
 ==== image:images/yes.png[yes] RPM (noun)
-*Description*: _RPM_ is the recursive initialism for the _RPM Package Manager_. RPM manages files in the RPM format, known as RPM packages. RPM packages are known informally as `rpm` files, but this informal usage is not used in Red Hat documentation to avoid confusion with the command name. Files in RPM format are referred to as "RPM packages".
+*Description*: _RPM_ is the recursive initialism for the _RPM Package Manager_. RPM manages files in the RPM format. Those files are referred to as "RPM packages". Even though RPM packages are known informally as `rpm` files, do not use this informal term in Red Hat documentation to avoid confusion with the command name.
```

Comment:
i'm sure i'm not the only one of us who's bothered by "RPM" standing for "RPM Package Manager," bc of the obvious redundancy. worth it to add "Originally known as Red Hat Package Manager" somewhere?

also, consider this change (for easier readability): "...Even though..., to avoid confusion with the command name, do not use this informal...."/?

Reply:
RPM already discussed in one of the comments above